cfme/tests/middleware/test_middleware_datasource.py

```diff
@@ -14,6 +16,13 @@
 ITEMS_LIMIT = 2  # when we have big list, limit number of items to test
 
 
+@pytest.yield_fixture(scope="function")
+def oracle_database():
```

Comment:
what is wrong here in using fixture as a setup/teardown for database resource? @psav wdyt?